### PR TITLE
feature/FM-01-new-catalog-nav-filter-category

### PR DESCRIPTION
**Pull request by DanielMino19**
sub x sub

### CODE REVIEW

File: src/components/catalog/ShopNavComponentNew.tsx [hunk 1]
```diff
@@ -48,7 +53,8 @@ export const ShopNavComponentNew = ({ handleFilterNav }: any) => {
         </div>
         <ul className="agri-ul widget-wrapper">
           <li>
-            <a href="#" className="d-flex flex-wrap justify-content-between">
+            <a href="#" className="d-flex flex-wrap justify-content-between"
```
Comment (davidcarodsn) [inline src/components/catalog/ShopNavComponentNew.tsx:56]: We try to replace any href="#" for type="button"

File: src/components/catalog/ShopNavComponentNew.tsx [hunk 2]
```diff
@@ -58,7 +64,8 @@ export const ShopNavComponentNew = ({ handleFilterNav }: any) => {
           { mainNavData.map(data => {
             return (
               <li>
-                <a href="#" className="d-flex flex-wrap justify-content-between">
+                <a href="#" className="d-flex flex-wrap justify-content-between"
```
Comment (davidcarodsn) [inline src/components/catalog/ShopNavComponentNew.tsx:67]: Same here

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,8 @@
+EMAILJS_PUBLIC_KEY=xxxx
+EMAILJS_SERVICE=xxxx
+
+FOOTER_EMAILJS_TEMPLATE=xxxx
+CONTACT_EMAILJS_TEMPLATE=xxxx
+
+RECAPTCHA_KEY=xxxx
+RECAPTCHA_SECRET_KEY=xxxx
```
Comment (davidcarodsn) [inline .env.example]: We lost some keys here, need to complete it

File: src/components/catalog/ShopNavComponentNew.tsx (hunk 1 above)
Comment (DanielMino19) [reply src/components/catalog/ShopNavComponentNew.tsx:56]: this href is needed for travell top site to user when him clicks it

**Comment by davidcarodsn** [comment]
LGTM / Ok / You rocks!

**Pull request merged**